### PR TITLE
Trainee dashboards

### PR DESCRIPTION
**Pull request by aditipate**
**This PR adds the following dashboards:**

- **Dash Manhattan Plot dashboard**
This dashboard utilizes the Plotly Dash Bio Suite to present an interactive Manhattan plot, allowing users to explore GWAS data. It includes a slider to modify the Genome Wide Significance threshold and a Dash table to display top SNPs and their associated gene/gene summaries.

- **Streamlit Stroke Predictor dashboard**
This dashboard includes a stroke predictor form containing fields for determinants of stroke. A user can provide this data for a hypothetical individual, which are then used by a classifier to predict the likelihood of the individual experiencing stroke.

- **Streamlit Image Comparison dashboard**
This dashboard allows users to select a pair of images consisting of a raw and flat field corrected image. Images are displayed side by side using the Streamlit Image Comparison component so that they may be compared by adjusting a slider, overall enhancing the ability to visualize differences.

- PR also includes READMEs for each of the dashboards
- Sample data needs to be added to Streamlit Image Comparison dashboard

### CODE REVIEW

File: tutorials/dash/dash-manhattan-plot/app.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+
+def parse(string):
+    parts = string.split('<br>')
+    snp = parts[0].split(': ')[1]
+    gene = parts[1].split(': ')[1]
+    return snp, gene 
+
+def get_ncbi_sum(gene_name):
+    Entrez.email = " please provide your email here "
```
Comment (ktaletsk) [inline tutorials/dash/dash-manhattan-plot/app.py:19]: Use `JUPYTERHUB_USER` env variable. It is going to contain user's

Comment (aditipate) [reply tutorials/dash/dash-manhattan-plot/app.py]: **Branch updates:**

* Updated Dash example to use `JUPYTER_USER` env variable to retrieve user's email
* Also updated README to remove email instructions

**Comment by aditipate** [comment]
**Additional update:**

- Added verification and validation for user email

File: tutorials/dash/dash-manhattan-plot/app.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+import pandas as pd
+from flask import Flask
+from dash.dependencies import Input, Output
+from dash import Dash, html, dcc
+import dash_bio as dashbio
+from dash import dash_table
+from dash.dash_table.Format import Group
+from Bio import Entrez
+from xml.etree import ElementTree as ET
+import re
```
Comment (ktaletsk) [inline tutorials/dash/dash-manhattan-plot/app.py]: Built-in dependencies are imported in the top

File: tutorials/dash/dash-manhattan-plot/app.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    gene = parts[1].split(': ')[1]
+    return snp, gene 
+
+def get_ncbi_sum(gene_name):
+
+    user_email = os.environ.get('JUPYTERHUB_USER')
+
+    entrez_email = verify_email(user_email)
+
+    Entrez.email = entrez_email
```
Comment (ktaletsk) [inline tutorials/dash/dash-manhattan-plot/app.py:35]: You can save one variable allocation and combine these two lines in one

**Review by ktaletsk — CHANGES_REQUESTED** [review]

File: tutorials/dash/dash-manhattan-plot/app.py (hunk 3 above)
Comment (aditipate) [reply tutorials/dash/dash-manhattan-plot/app.py]: Got it! Just updated and combined the lines

**Review by ktaletsk — APPROVED** [review]
Congratulations on your first PR merged into PolusAI!

**Pull request merged**